metadata/metadata_tutorial_addition.Rmd

```diff
@@ -0,0 +1,622 @@
+date: "4/30/2019"
+output: html_document
+---
+
+# Section 1: Overview
+
+The objective of this tutorial is to demonstrate how to upload a season's metadata to the traits database. All traits data need to have associated metadata.
+
+Each season should have metadata for experiments, sites, treatments, cultivars, and citations.
+
+For this tutorial, you will be uploading your metadata to a test database. Instructions on how to set up a local instance of a test database can be found [here](https://gist.github.com/kimberlyh66/935106e35478829b9e844ba5b496784d).
+
+This tutorial will only cover how to add new experiments, sites, treatments, cultivars, and citations. 
+
+The `Metadata Upload Tutorial Part 2: Associations` tutorial should be viewed after you have completed this tutorial and all new metadata have been added (see below for more details on how to run this markdown file). The second tutorial will cover how to associate the metadata that you add.
+
+# Section 2: Required input
+
+URL to a public published google sheet (you will need to provide this url below in section 4)
+
+   * copy and fill out the following [google sheets template](https://docs.google.com/spreadsheets/d/1pNEuwNHNHHrpql0b36DYFAbOlBU5TJVF_nO4RdRzdmg/edit#gid=279516453).
+   * instructions on how to fill out the template can be found under the README worksheet.
+   * make the google sheet public (to anyone with link) and publish the entire document to the web as a xlsx.
+
+# Section 3: How to Run this Rmd
+
+## Commit database transactions to upload data
+
+Knitting this markdown file will not upload any data to the test database. Being able to knit this file means that you will not encounter any errors when uploading your metadata. All experiments, sites, treatments, cutlivars, and citations can be uploaded successfully. 
+
+To actually commit additions to the test database, you will need to make an edit to this markdown file. 
+
+The value of `commit` needs to be changed from `FALSE` to `TRUE` in the following chunk:
+
+```{r commit-param}
+
+commit <- FALSE
+
+```
+
+It is important to note that you should only be uploading your metadata once. 
+
+Trying to upload your metadata more than once will result in errors. 
+
+If you would like to reinitialize the test database and remove any additions that you have made, re-run step 3 of the `How to set up local instance of BETY test database` tutorial.
+
+## Check uploaded data
+
+You will not be able to fetch any of your uploaded data until it has been added to the database. 
+
+You can check your uploaded results when you run this Rmd with `commit <- TRUE` in the above chunk. 
+
+The value of `eval_chk` will depend on the value you set for `commit`. This variable will determine whether chunks for checking uploaded data will be evaluated. You will not see any tables below these chunks unless you set commit to TRUE. 
+
+```{r check-upload, results = 'hide'}
+
+ifelse(commit == TRUE, eval_chk <- TRUE, eval_chk <- FALSE)
```

Comment:
either set it this way:

```suggestion
eval_chk <- commit
```

or just use `eval = commit` in the chunk arguments.